web-animations-1/Overview.bs

```diff
@@ -459,39 +459,41 @@ Specification conventions {#spec-conventions}
 Web Animations model overview {#web-animations-model-overview}
 =============================
 
-<div class=informative-bg>
-<em>This section is non-normative</em>
-
   At a glance, the [=Web Animations model=] consists
   of two largely independent pieces:
-  a <em>timing model</em> and an <em>animation model</em>.
+  a [=timing model=] and an [=animation model=].
   The roles of these pieces are as follows:
 
-  : Timing model
+  : <dfn id="timing-model-dfn">timing model</dfn>
   ::
       Takes a moment in time and converts it to a proportional distance
       within a single iteration of an animation,
-      called the <em>iteration progress</em>.
-      The <em>iteration index</em> is also recorded
+      called the <dfn>iteration progress</dfn>.
+      The <dfn>iteration index</dfn> is also recorded
```

Comment:
This should link to the definition of the current iteration.

Reply:
OK, I didn't catch that they're the same concept. In that case, can we fold the names together and use “iteration index” throughout?

Reply:
It's exposed in the API as `currentIteration` so can we use "current iteration" instead? Or "current iteration index"?

Reply:
Sure, works for me. I'll try to layer that in then.